libr/core/core.c

```diff
@@ -1276,7 +1277,11 @@ R_API int r_core_init(RCore *core) {
 
 	r_core_config_init (core);
 
+	ut64 diff, prev = r_sys_now();
 	r_core_loadlibs_init (core);
+	diff = r_sys_now() - prev;
+	sdb_num_add(DB, "r_core_loadlibs_init time", diff, 0);
```

Comment:
Maybe is better to store the diff in a variable inside RCore and compute this time diffing in radare2.c

But this will only work from r2, not any other program using rcore.

What im thinking is that what we need to benchmark here are two things:

* plugins load time
* file load time

I think we should have two variable in rcore for that. Instead of using sdb and saving the difftimes for those two cases. This info must be displayed with the ?T command

We can add an RCoreTimes struct with all the time fields we are interested in. 

> On 16 Mar 2016, at 22:37, Ahmed Mohamed Abd El-MAwgood <notifications@github.com> wrote:
> 
> In libr/core/core.c:
> 
> >  	r_core_loadlibs_init (core);
> > +	diff = r_sys_now() - prev;
> > +	sdb_num_add(DB, "r_core_loadlibs_init time", diff, 0);
> maybe it should be r_core_loadlibs_init_time I am not sure of if the command line will be able to handle queries with space in the name @radare
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>